distributed/tests/test_channels.py

```diff
@@ -202,3 +202,32 @@ def test_values(c, s, a, b):
     assert list(x2.data) == data
 
     yield c2._shutdown()
+
+
+def test_channel_gets_updates_immediately(loop):
+    with cluster() as (s, [a, b]):
+        with Client(s['address']) as c:
+            x = c.channel('x')
+            future = c.submit(inc, 1)
+            x.append(future)
+            x.flush()
+
+        with Client(s['address']) as c:
+            x = c.channel('x')
+            future = next(iter(x))
+            assert future.result() == 2
+
+
+def test_channel_gets_updates_immediately_2(loop):
+    with cluster() as (s, [a, b]):
+        with Client(s['address']) as c:
+            x = c.channel('x')
+
+            with Client(s['address']) as c2:
+                x2 = c.channel('x')
```

Comment:
I might misunderstand the intent here, but do `x2` and `c2` actually have a use in this test?